topological_navigation/src/topological_navigation/edge_action_manager.py

```diff
@@ -57,7 +57,7 @@ class EdgeActionManager(object):
     def __init__(self):
         
         self.client = None        
-        self.current_action = None
+        self.current_action = "none"
```

Comment:
Are you sure about this? `None` is quite different to "none"` 
Are we checking `self.current_action` anywhere?

Reply:
Doesnt make a difference. The preempt function checks if the client is not None and if the client is not None then the current action is also not "none". Have you tested this? Does it break?

Reply:
So self.action name is not checked explicitly, it is checked by checking that the client is not None